scripts/utils.ts

```diff
@@ -0,0 +1,5 @@
+import type { EventLog, Log } from "ethers";
```

Comment:
the response from `receipt?.logs` (see above) is now a `Log` or `EventLog`, so we need to make sure we only get the `EventLog` in order to parse